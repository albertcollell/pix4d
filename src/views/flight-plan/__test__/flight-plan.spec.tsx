import { shallow } from 'enzyme';
import React from 'react';
import * as enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import FlightPlan from '../flight-plan';

enzyme.configure({ adapter: new Adapter() });

describe('FlightPlan', () => {
  it('renders FlightPlan without crashing', () => {
    const wrapper = shallow(<FlightPlan />);
    expect(wrapper).toMatchSnapshot();
  });
});
