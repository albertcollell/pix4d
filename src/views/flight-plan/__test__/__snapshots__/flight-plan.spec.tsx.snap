// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlightPlan renders FlightPlan without crashing 1`] = `ShallowWrapper {}`;
